### PR TITLE
Add edit observation refinements

### PR DESCRIPTION
**Pull request by AlexeyKaryka**

### CODE REVIEW

File: src/screens/AddEditObservation/AddObservation.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+const AddObservation = props => {
+  return (
+    <ObservationBase
+      {...props}
+      submitButtonText={translate("addEditObservation.sendObservation")}
+    />
+  );
+};
+AddObservation.navigationOptions = ({ navigation }) => {
```
Comment (elliolir) [inline src/screens/AddEditObservation/AddObservation.js]: from the readability point of view, it looks like a hack to me 🙃 maybe it makes sense to make it regular static field of the class?

Comment (AlexeyKaryka) [reply src/screens/AddEditObservation/AddObservation.js]: I agree with you, will update to class component

Comment (AlexeyKaryka) [reply src/screens/AddEditObservation/AddObservation.js]: FIxed

File: src/screens/AddEditObservation/ObservationBase.js [hunk 2]
```diff
@@ -89,9 +91,23 @@ const AddEditObservation = props => {
     props.onCurrentDateTime();
   };
 
+  const onCancelExit = () => {
```
Comment (AfanasyShevtsov) [inline src/screens/AddEditObservation/ObservationBase.js]: I would prefer `onCancel` instead of `onCancelExit`. Because we already have `onExit`.

Comment (AlexeyKaryka) [reply src/screens/AddEditObservation/ObservationBase.js]: FIxed

**Pull request merged**